bin/MadGraph5_aMCatNLO/cards/production/13TeV/HeavyNeutrino_trilepton_V-0.01_tau_NLO/HeavyNeutrino_trilepton_mNgtmWltmZ_V-0.01_tau_NLO/HeavyNeutrino_trilepton_V-0.01_tau_NLO_proc_card.dat

```diff
@@ -0,0 +1,25 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = ta+
+define ldecay+ = e+ mu+ ta+
+define l- = ta-
+define ldecay- = e- mu- ta-
+define l = l+ l-
+define ldecay = ldecay+ ldecay-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+define v = vl vl~
+define w = w- w+
+import model SM_HeavyN_NLO
+define p = p b b~
+define j = p
+generate p p > n1 l [QCD]
```

Comment:
Can you change this to "generate p p > l n1 [QCD]" with reasons in [1]?

[1] https://answers.launchpad.net/mg5amcnlo/+question/661535